README.md

```diff
@@ -46,66 +46,63 @@ val newAddress = wallet.getNewAddress()
 ### Example Projects
 
 #### `bdk-android`
-
 * [Devkit Wallet](https://github.com/thunderbiscuit/devkit-wallet)  
 * [Padawan Wallet](https://github.com/thunderbiscuit/padawan-wallet)
 
 #### `bdk-jvm`
-
 * [Tatooine Faucet](https://github.com/thunderbiscuit/tatooine)
 
 ### How to build
 _Note that Kotlin version `1.6.10` or later is required to build the library._
 
-1. Clone this repository and init and update it's [`bdk-ffi`] submodule.
-   ```shell
-   git clone https://github.com/bitcoindevkit/bdk-kotlin
-   git submodule update --init
-   ```
-1. Follow the "General" bdk-ffi ["Getting Started (Developer)"] instructions.
-1. If building on MacOS install required intel and m1 jvm targets
-   ```sh
-   rustup target add x86_64-apple-darwin aarch64-apple-darwin
-   ```
-1. Install required targets
-    ```sh
-    rustup target add x86_64-linux-android aarch64-linux-android armv7-linux-androideabi i686-linux-android
-    ```
-1. Install `uniffi-bindgen`
-    ```sh
-    cargo install uniffi_bindgen --version 0.16.0
-    ```
-    See the [UniFFI User Guide](https://mozilla.github.io/uniffi-rs/) for more info
-1. Install Android SDK and Build-Tools for API level 30+
-1. Setup `$ANDROID_SDK_ROOT` and `$ANDROID_NDK_ROOT` path variables (which are required by the 
-   build scripts), for example (NDK major version 21 is required):
-    ```shell
-    export ANDROID_SDK_ROOT=~/Android/Sdk
-    export ANDROID_NDK_ROOT=$ANDROID_SDK_ROOT/ndk/21.<NDK_VERSION>
-    ```
-1. Build kotlin bindings
-    ```sh
-    ./build.sh
-    ```
-1. Start android emulator and run tests
-   ```sh
-   ./gradlew connectedAndroidTest 
-   ```
+1. Clone this repository and initialize and update its [`bdk-ffi`] submodule.
+```shell
+git clone https://github.com/bitcoindevkit/bdk-kotlin
+git submodule update --init
+```
+2. Follow the "General" bdk-ffi ["Getting Started (Developer)"] instructions.
+3. If building on MacOS install required intel and m1 jvm targets
+```sh
+rustup target add x86_64-apple-darwin aarch64-apple-darwin
+```
+4. Install required targets
+ ```sh
+ rustup target add x86_64-linux-android aarch64-linux-android armv7-linux-androideabi
+ ```
+5. Install Android SDK and Build-Tools for API level 30+
+6. Setup `$ANDROID_SDK_ROOT` and `$ANDROID_NDK_ROOT` path variables (which are required by the 
+   build tool), for example (NDK major version 21 is required):
+ ```shell
+ export ANDROID_SDK_ROOT=~/Android/Sdk
+ export ANDROID_NDK_ROOT=$ANDROID_SDK_ROOT/ndk/21.<NDK_VERSION>
+ ```
+7. Build kotlin bindings
+ ```sh
+ # build JVM library
+ ./gradlew :jvm:buildJvmLib
+ 
+ # build Android library
+ ./gradlew :jvm:buildAndroidLib
+ ```
+8. Start android emulator and run tests
```

Comment:
Worth adding a reminder here to use x86_64 version of emulator. I found today while testing I was still using an x86 version of the emulator and took me some time (too long) to figure it out. :face_with_spiral_eyes: 
```suggestion
8. Start android emulator (must be x86_64) and run tests
```

Reply:
Were you on your mac M1 or on Linux? Potentially related to #52. If you _are_ using `x86` I'm not opposed to supporting it... I just _think_ it's not very much used, but let me know if you think otherwise.

Reply:
I was on my linux box, but I guess could have been a problem on my mac if I'd been using the x86 emulator.